BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+cc_library(
+    name = "json",
+    hdrs = glob([
+        "include/nlohmann/adl_serializer.hpp",
+        "include/nlohmann/byte_container_with_subtype.hpp",
```

Comment:
This is not maintainable (and `glob()` is unnecessary). Please generate `BUILD.bazel` from CMake as suggested in the linked PR.